src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -95,34 +87,62 @@ public void setup() throws InterruptedException, ServiceBusException, JSONExcept
             .atMost(60, TimeUnit.SECONDS)
             .pollDelay(2, TimeUnit.SECONDS)
             .until(() -> isExceptionRecordCreated(randomPoBox));
+    }
 
+    @Test
+    public void should_attach_exception_record_to_the_existing_case_with_no_evidence() {
+        //given
+        CaseDetails caseDetails = createCase("envelopes/new-envelope.json");
+
+        // when
+        invokeCallbackEndpointForLinkingDocsToCase(caseDetails, exceptionRecord);
+
+        //then
+        await("Exception record is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> isExceptionRecordAttachedToTheCase(caseDetails, 1));
+
+        verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord);
     }
 
     @Test
-    @Disabled
-    public void should_atatch_exception_record_to_the_existing_case() {
+    public void should_attach_exception_record_to_the_existing_case_with_evidence_documents() throws JSONException {
+        //given
+        Envelope newEnvelope = updateEnvelope("envelopes/new-envelope-with-evidence.json");
+        CaseDetails caseDetails = ccdCaseCreator.createCase(newEnvelope);
```

Comment:
better but the newEnvelope is always the same ... why not build it in make it static and re-use it in other tests :D